### ISSUE TITLE
The handling of fractions is completely broken

### ISSUE THREAD

**Issue by jackmaney** [gogotanaka/hilbert#78]
```
Hi guys,thank you for using Hilbert.
You need to execute "postulate zfc_analysis" if you wanna do real analysis.

Enjoy! -> postulate zfc_analysis
success! :)
Enjoy! -> d/dx x^(1/2)
0
Enjoy! -> d/dx x^(1/3)
0
Enjoy! -> d/dx x^0.5
0.5( x ^ -0.5 )
```

**Comment by jackmaney** [comment]
It's far, far worse than I thought:

```
Hi guys,thank you for using Hilbert.
You need to execute "postulate zfc_analysis" if you wanna do real analysis.

Enjoy! -> postulate zfc_analysis
success! :)
Enjoy! -> 1/2 + 1/2
0.0
```

**Comment by gogotanaka** [comment]
@jackmaney Always thank you so much. I'm really appreciate your contribute, I have to say sorry for solving quite few your great reports so far. 

Actually I'm applying for government programs to develop Hilbert. If I pass it, I can develop Hilbert with salary at full-time! Please wait just a little longer.

Aside from that, I've solved this issue at [dec5b19](https://github.com/gogotanaka/Hilbert/commit/dec5b198fd9235a0c4bdf45f8dc441da83dd0c9c), and released as v0.0.2700320.

p.s. I forgot to add tests, https://github.com/gogotanaka/Hilbert/commit/e960effeef516f06ad75663d8584e92d354e132f

Thanks!
